swift/ql/lib/codeql/swift/controlflow/CfgNodes.qll

```diff
@@ -142,3 +142,7 @@ class KeyPathApplicationExprCfgNode extends ExprCfgNode {
 class KeyPathExprCfgNode extends ExprCfgNode {
   override KeyPathExpr e;
 }
+
+class EntryNode = Impl::EntryNode;
+
+class ExitNode = Impl::ExitNode;
```

Comment:
We also need `AnnotatedExitNode`, I think (because it was also part of the public interface before these changes).